packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSetInlineSource.js

```diff
@@ -82,18 +87,14 @@ export function completeBoundaryWithStyles(
           resourceEl.setAttribute(attr, stylesheetDescriptor[j++]);
         }
         loadingState = resourceEl['_p'] = new Promise((resolve, reject) => {
-          resourceEl.onload = resolve;
-          resourceEl.onerror = reject;
+          resourceEl.onload = cleanupWith.bind(resourceEl, resolve);
```

Comment:
It might be smaller to just inline the function like an arrow function. We probably compile them out but if we ever stop.

Reply:
need bind b/c i reuse the binding. not doing so leads to closure spitting out very large code b/c it gives the variable some enormous loop name